### DISCUSSION TITLE
A possible issue with the handleError chapter of the SvelteKit tutorial

### DISCUSSION THREAD

**Discussion by brucegl** [sveltejs/kit#11872] [Q&A] [answered]
Hi,

In [https://learn.svelte.dev/tutorial/handleerror](https://learn.svelte.dev/tutorial/handleerror), is the "solve" solution meant to handle the unexpected error gracefully when you navigate to "the bad place"? Nothing happens for me.... maybe I misunderstood something.

Thanks
Gary

**Comment by teemingc** [comment, marked as answer]
I've converted this discussion into an issue in the learn.svelte.dev repo https://github.com/sveltejs/learn.svelte.dev/issues/586
